jenkins-pipelines/longevity-100gb-4h-ldap-authorization-authenticator-ms-ad.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    aws_region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: '''["test-cases/longevity/longevity-100gb-4h.yaml", "configurations/ms-ad-ldap-authorization-and-authentication.yaml"]''',
```

Comment:
IIUC this file, `configurations/ms-ad-ldap-authorization-and-authentication.yaml` can be replaced by `configurations/ms-ad-ldap-authenticator.yaml` and have a single MS AD with authentication

Reply:
I misunderstood of the 'use_ms_ad_ldap', we expected ldap authorization isn't enabled in configurations/ms-ad-ldap-authenticator.yaml.